data/vector2.lean

```diff
@@ -97,6 +97,11 @@ def {u} mmap {m} [monad m] {α} {β : Type u} (f : α → m β) :
   do h' ← f a, t' ← mmap f v, pure (h' :: t')
 | _ ⟨l, rfl⟩ := rfl
 
+theorem ext : ∀ {v w : vector α n}
```

Comment:
please add a `extensionality` attribute (and make sure it works)

Reply:
`vector.eq` already has the `extensionality` attribute for `vector`s